templates/pyenv.sh

```diff
@@ -1,12 +1,15 @@
-# Prefer a user pyenv over a system wide install
+# Prefer a user pyenv over a system-wide install
 if [ -s "${HOME}/.pyenv/bin" ]; then
     pyenv_root="${HOME}/.pyenv"
+    pyenv_init="pyenv init -"
 elif [ -s "<%= @global_prefix %>" ]; then
     pyenv_root="<%= @global_prefix %>"
     export PYENV_ROOT=${pyenv_root}
+    # Rehashing will fail in a system install
+    pyenv_init="pyenv init - --no-rehash"
 fi
 
 if [ -n "$pyenv_root" ]; then
     export PATH="${pyenv_root}/bin:$PATH"
-    eval "$(pyenv init -)"
+    eval "$($pyenv_init)"
```

Comment:
There's no `else` statement above so it won't do anything in that case. is that expected?

Reply:
yes.  Do you have a suggested action?

Reply:
PR #52 was sitting there in approved status for a long time. I'm trying to get the change merged.  I really don't have any opinion about the action to be taken, I'm trying to clean up the cruft, things like tickets that just need a rebase before they get merged or that don't execute tests.  You'll have a much better feel for what should happen in this situation.  Thanks

Reply:
oh nevermind, it was like this before. please disregard my comment.
it's been quite a while since I wrote this so forgot about it.